modules/sublimeBidAdapter.js

```diff
@@ -127,21 +129,24 @@ function buildRequests(validBidRequests, bidderRequest) {
 
     // Register a callback to send notify
     window[callbackName] = (response) => {
+      function querify(params) {
+        Object.keys(params).map(function (key) {
```

Comment:
Should add `return`, seems always return `undefined` ?

Reply:
Thanks!